### DISCUSSION TITLE
百度网盘不能上传文件

### DISCUSSION THREAD

**Discussion by zh1963678974** [AlistGo/alist#3825] [General]
[挂载在本地的百度网盘无法上传，提示目标文件夹访问被拒绝-你需要权限来执行此操作
<img width="627" alt="微信图片_20230312200249" src="https://user-images.githubusercontent.com/38040246/224543443-5fc953ee-3109-44cf-9dc8-9c2dfa99bdf5.png">

**Comment by bihvdgj** [comment]
我115网盘也是显示这个，有大佬知道怎么解决吗？

**Comment by anwen-anyi** [comment]
百度为什么不能上传 去常见问题自己翻

**Reply by xiaohan815** [reply]
为啥啊?是得用v2版本吗?

**Reply by lin-cop** [reply]
如果你不会设置 "User-Agent" 请看这里
以下方法仅限于有百度超级会员用户使用
再次提示 以下方法仅限于有百度超级会员用户使用
有会员改完 "User-Agent" 才会有用（选择官方和302）
如果不改 "User-Agent"，可以开启 Web代理，缺点是需要你搭建Alist的机器中转，也就是说你需要大宽带帮你中转

[https://alist.nn.ci/zh/guide/drivers/baidu.html](https://alist.nn.ci/zh/guide/drivers/baidu.html)

**Reply by xiaohan815** [reply]
这个我试过了,v3跟v2版本都能开启web代理,但是v3版本的不能上传文件啥的,我用了v2版本,发现ok了,v2的好几个网盘都正常,除了阿里云的
